Dockerfile

```diff
@@ -1,22 +1,42 @@
-FROM node:alpine AS base
-RUN mkdir -p /code
-WORKDIR /code
-CMD "hsd"
+FROM node:10-alpine AS base
 
-RUN apk upgrade --no-cache && \
-    apk add --no-cache bash unbound-dev gmp-dev
+RUN mkdir /opt/hsd
+WORKDIR /opt/hsd
 
-COPY package.json \
-     #package-lock.json \
-     /code/
+# dynamically linked deps
```

Comment:
Do we want dynamic deps? I thought the idea of keeping package.json here was to cache the builds so that if we don't update any dependencies we can skip the install step.

Reply:
These are native dependencies that have to be there during the build and at runtime. They won't change unless its a forced rebuild and `apk upgrade` finds a new version

Reply:
@tynes since you are using the default node10 alpine image, best practices recommends using the package versions that the image comes with instead of running apk upgrade.  This allows the maintainer to test the image and the pkg versions before releasing it.  Recommend pinning the version numbers to the below command and also removing the `apk upgrade` command.